main.py

```diff
@@ -0,0 +1,30 @@
+from arguments import parameters
+from keys import check_keys
+from requester import request_weatherapi, request_openmeteo, request_meteostat
+
+# get parameters
+param = parameters()
+print(param)
+
+# get API keys from the file
+keys = check_keys()
+print(f"The API keys have correct names:\n{list(keys.keys())}")
+
+# request to wheatherapi
+weatherapi_forecast = request_weatherapi(keys, param)
+print(weatherapi_forecast)
+latitude = weatherapi_forecast['latitude']
+longitude = weatherapi_forecast['longitude']
+current_date = weatherapi_forecast['hours'][1].split(' ')[0]
```

Comment:
This could be extracted to a separate function for better readability.

Reply:
Done. I acquired current_date totally independent from weatherapi, because it is cleaner and might be handy in the future. As for the longitude and latitude I just changed the output of the request_weatherapi() function, to generate the values. I even didn't have to change a lot, because the function was already generating these variables I just had to add them to the 'return' line.